priv/Dockerfile.release

```diff
@@ -1,15 +1,10 @@
-FROM bitwalker/alpine-erlang:6.1
+FROM alpine:3.5
 
-RUN apk update && \
-    apk --no-cache --update add libgcc libstdc++ && \
-    rm -rf /var/cache/apk/*
+RUN apk --no-cache add ncurses-libs
 
 EXPOSE 4000
-ENV PORT=4000 MIX_ENV=prod REPLACE_OS_VARS=true SHELL=/bin/sh
+ENV PORT=4000 MIX_ENV=prod REPLACE_OS_VARS=true
 
 ADD ${APP}.tar.gz ./
-RUN chown -R default ./releases
-
-USER default
```

Comment:
Something like this perhaps -> https://github.com/lpil/progress/blob/master/rel/Dockerfile.release